lib/Models/LineParameter.js

```diff
@@ -64,4 +64,15 @@ LineParameter.prototype.formatValueAsString = function(value) {
     }
 };
 
+LineParameter.prototype.getGeoJsonFeature = function(value) {
```

Comment:
Strange that this is an instance method but it doesn't ever use `this`.  Should it at least assume the current value of the parameter if one isn't supplied?